diagnostics/ENSO_RWS/doc/ENSO_RWS.rst

```diff
@@ -0,0 +1,294 @@
+**Rossby Wave Sources Diagnostic Package**
+
+**Last update 04/15/2021**
+
+ENSO Rossby wave sources (ENSO_RWS) diagnostic package consists of four
+levels. With a focus on identifying leading processes that determine
+ENSO-induced global teleconnection, particularly the Pacific North
+American (PNA) pattern, main module of the POD estimates basic state
+flow properties at an appropriate tropospheric upper-level and solves
+barotropic vorticity equation to estimate various terms that contribute
+to the total anomalous RWS. In that pursuit, POD is applied to monthly
+data (climate model or reanalysis products), and RWS terms are estimated
+for “composite” El Niño or La Nina events. To attain robust “composite”
+results a reasonable sample of ENSO winters is needed. However, the POD
+can be applied even for a single El Niño winter (e.g., when applied to
+seasonal prediction models). Similarly, the POD is applicable to any
+number of pressure levels (e.g., to identify the level at which maximum
+upper-level divergence and associated RWS are located). Here, reanalysis
+products (e.g., ERA-interim) is “considered” as “observations” and
+diagnostics obtained from ERA-Interim and other reanalysis products are
+used for model validation. In this general document, brief descriptions
+of the four levels of the POD are provided but detailed information is
```

Comment:
brief descriptions
of the four levels of the POD are provided[,] ~~but~~ and detailed information is

Reply:
edited